### PR TITLE
DOC: Strip outdated extension list and refer to handbook

### PR DESCRIPTION
**Pull request by mih**
The Handbook URL anticipates a change in the handbook that will come
shortly. https://github.com/datalad-handbook/book/pull/484

Fixes gh-4490

### CODE REVIEW

**Comment by mih** [comment]
Arrgh, `GitRepo.get_remotes()` obviously doesn't want ...

File: README.md [hunk 1]
```diff
@@ -27,13 +27,9 @@ A number of extensions are available that provide additional functionality for
...
-automatically installed with it. Here is a list of known extensions:
-
-- [crawler](https://github.com/datalad/datalad-crawler) -- tracking web resources and automated data distributions [![crawler release](https://img.shields.io/github/release/datalad/datalad-crawler.svg)](https://GitHub.com/datalad/datalad-crawler/releases/)
-- [neuroimaging](https://github.com/datalad/datalad-neuroimaging) -- neuroimaging research data and workflows [![neuroimaging release](https://img.shields.io/github/release/datalad/datalad-neuroimaging.svg)](https://GitHub.com/datalad/datalad-neuroimaging/releases/)
-- [container](https://github.com/datalad/datalad-container) -- support for containerized computational environments [![container release](https://img.shields.io/github/release/datalad/datalad-container.svg)](https://GitHub.com/datalad/datalad-container/releases/)
-
-- [webapp](https://github.com/datalad/datalad-webapp) -- support for exposing selected DataLad API as REST API webapp [tech demo]
+automatically installed with it. An [annotated list of available
+extensions](http://handbook.datalad.org/extension_pkgs.html) is available in the
+[DataLad handbook](http://handbook.datalad.org).
```
Comment (kyleam) [inline README.md:32]: Looks fine to me, though "... list of _available_ extensions is _available_" is a little clunky.  Perhaps just drop the first "available"?

Comment (mih) [reply README.md:32]: Will do, thx!

Comment (mih) [reply README.md]: Done

**Comment by yarikoptic** [comment]
Let it be so

**Pull request merged**